README.md

```diff
@@ -23,8 +23,13 @@ npm install -g yo
 # install the react-server generator
 npm install -g generator-react-server
 
-# make a new react-server project
+# make a new react-server project in the CURRENT directory
```

Comment:
yup, that's just how yeoman works.  I guess I didn't think to add it to the README because its also what I expect; change lgtm